rbd/snapshot_test.go

```diff
@@ -76,6 +76,9 @@ func TestErrorSnapshotNoName(t *testing.T) {
 	snapshot, err := img.CreateSnapshot("")
 	assert.NoError(t, err)
 
+	err = img.SetSnapshot(snapshot.name)
```

Comment:
It would probably better if we replaced the use of `snapshot.Set`  at around line number 97.